apps/balancer-tools/src/app/apr/pool/(components)/PoolOverviewCards.tsx

```diff
@@ -21,20 +21,22 @@ async function AverageAPRCard({ poolId }: { poolId: string }) {
 }
 
 export default async function PoolOverviewCards({
-  roundId,
+  startAt,
+  endAt,
   poolId,
 }: {
-  roundId?: string;
+  startAt: Date;
+  endAt: Date;
   poolId: string;
 }) {
   const cardsDetails: {
     title: string;
     content: JSX.Element | string;
     tooltip?: string;
   }[] = [];
-  if (roundId) {
+  if (startAt || endAt) {
     const results: PoolStatsResults = await fetcher(
-      `${BASE_URL}/apr/api/?poolId=${poolId}&sort=roundId`,
+      `${BASE_URL}/apr/api/?poolId=${poolId}&startAt${startAt}&endAt=${endAt}`,
```

Comment:
here we'd probably not want to set `endAt=undefined` or would we?

Reply:
If start/end is undefined the client is redirected to home